src/pages/patterns/fluid-variants/index.mdx

```diff
@@ -0,0 +1,200 @@
+| Fixed width       | The width of an element is set to a specific value and does not change.                                                                                                                                                                                                    |
+| Fluid width       | The width of an element is responsive, often proportionally changing between breakpoints or is set as a percentage of a container.                                                                                                                                         |
+| Default variant   | The primary style and behavior of a component that may respond either in a fluid or fixed width manner depending on layout and responsive strategy. They are more productive in nature and are the most commonly used variant.                                             |
+| Fluid variant     | An alternative style and behavior available in some components that always has a fluid width. It is a percentage-based element because it comprises part of a greater whole. Fluid variants are more expressive and are used to emphasize an experience.                   |
+| Productive moment | A productive experience is all about keeping the user focused and able to complete complex tasks. Productive elements may appear more conservative, confident, and efficient.                                                                                              |
+| Expressive moment | An expressive experience creates a desired impact or pause in an experience. They are often used when users are learning, exploring, and interacting through impactful imagery, layout, and long-form reading. Expressive elements may evoke more personality and emotion. |
+
+<Row>
+<Column colLg={8}>
+
+![Example of fixed and fluid](/images/Layout_overview_Fluid-base-unit.svg)
+
+</Column>
+</Row>
+
+### When to use fluid variants
+
+- Use in expressive or important moments, to better capture the user’s attention
+  and offer a rhythmic break to the productive experience.
+- Use within or attached to compound components.
+- Use in spacious layouts where the fluid variants have space to expand and
+  breath.
+
+### When not to use fluid variants
+
+- Don’t use when white space is needed between input components or in dense
+  layout where smaller components may be preferred.
+- Don’t use in hyper productive moments like a complex form.
+- Don’t use inside accordions or similarly sectioned containers where the edge
+  of the fluid component align to the same edge as the accordion dividers
+  creating hierarchical confusion.
+
+## Types of fluid components
+
+Not all components come with a fluid variant but each fluid instance has a
+default counterpart.
+
+### Fluid inputs
+
+Fluid inputs are an alternative input style with the label placed inside the
```

Comment:
Fluid inputs are an alternative input style with the label placed inside the input field, aligning with the user input text. All text is contained inside the field, unlike the default input, where the label and helper text sit above and below (respectively) outside the field container. Fluid inputs only have one height, at 64px, but have a fluid width that hugs one or more edges of its container. See the individual input components for more guidance on each fluid option.